src/mixins/controls.js

```diff
@@ -146,8 +147,48 @@ const OrbitControlsMixin = (superclass) => class extends superclass {
         this.renderer2 = this.getWebGLRenderer({alpha: true});
         this.renderer2.setClearColor("#FFFFFF", 0);
         this.renderer2.setSize(containerDimension, containerDimension);
+				this.renderer2.domElement.style.position = 'absolute';
+				this.renderer2.domElement.style.right = '10px';
         this.container.prepend(this.renderer2.domElement);
 
+        //For axis labels
+				this.labelRenderer = new CSS2DRenderer();
+        this.labelRenderer.setSize(containerDimension, containerDimension);
+				this.labelRenderer.domElement.style.position = 'absolute';
+				this.labelRenderer.domElement.style.right = '10px';
+				this.labelRenderer.domElement.style.pointerEvents = 'none';
+        this.container.prepend(this.labelRenderer.domElement);
+
+        //axis label X
+				const textX = document.createElement( 'div' );
+				textX.className = 'label';
+				textX.style.color = 'rgb(255,255,255)';
+				textX.textContent = 'X';
+				const labelX = new CSS2DObject( textX );
+				labelX.position.x = length * 1.2;
+				labelX.position.y = 0;
+				labelX.position.z = 0;
```

Comment:
The above code on lines 163-170 is repeated 3 times - this is bad practice. Let's:

(1) implement labels.js as a mixin file
(2) isolate the label drawing functionality to a helper function - `addAxisLabel`, for example
(3) `have addAxisLabels` function that calls the (2) on X, Y, and Z
(4) add the (3) to `initSecondAxes` above